front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -499,7 +499,7 @@ export default function AssistantBuilder({
           })()}
           <PrevNextButtons screen={screen} setScreen={setScreen} />
         </div>
-        {false && <div className="flex flex-col space-y-8 pb-16 pt-8"></div>}
+        <div className="flex flex-col space-y-8 pb-16 pt-8"></div>
```

Comment:
why a div and not a spacing as class of the div above?
can have a look if needed :+1:

Reply:
It was added here: https://github.com/dust-tt/dust/commit/091bd229af538cce9370d4ab7a1a872dfeb609bb

Reply:
Ok will have a look later, you can merge in the meantime

Reply:
It should probably be part of the App Layout at some point 🙃.